app/views/agencies/show.html.erb

```diff
@@ -27,12 +24,6 @@
   <%= show_non_blank_fields "Email", @agency.email %>
 </p>
 
-<p>
-  <% if @agency.website.present? %>
-     Website: <%= link_to @agency.website, @agency.website %>
```

Comment:
@StlMaris123 You haven't actually made the change to the helper, just deleted this section.

Reply:
@StlMaris123 No, what I'm saying is that in line 13, you use the helper to display the jurisdiction if it's present.  I need you to do the same thing for the website - use the helper to display the website if it's present.  Replace lines 30-35 with that.